criu/pie/restorer.c

```diff
@@ -425,6 +425,32 @@ static int restore_signals(siginfo_t *ptr, int nr, bool group)
 	return 0;
 }
 
+static int restore_cpu_affinity(struct task_restore_args *args)
+{
+	int i;
+	int pid;
+	int ret;
+	cpu_set_t *cpumask;
+	cpu_set_t *allowed_cpus;
+
+	if (!args->with_cpu_affinity) {
```

Comment:
Is it a problem in case in dump we don't use `with_cpu_affinity` but in restore we use `with_cpu_affinity`? As far as I can see, the value of cpumask in core image is uninitialized when dumping without `with_cpu_affinity`.

Reply:
thanks, I add has_cpumask variable solves this problem.